Firebase/Storage/FIRStorageMetadata.m

```diff
@@ -180,6 +180,10 @@ - (NSDictionary *)dictionaryRepresentation {
   if (_path) {
     metadataDictionary[kFIRStorageMetadataName] = _path;
   }
+  
```

Comment:
This method has spaces for all of these blocks. I can remove them all if warranted.